src/ngraph/codegen/CMakeLists.txt

```diff
@@ -74,4 +70,9 @@ set_target_properties(codegen PROPERTIES VERSION ${NGRAPH_VERSION} SOVERSION ${N
 add_dependencies(codegen header_resource libmkldnn libeigen)
 target_include_directories(codegen SYSTEM PUBLIC ${CMAKE_BINARY_DIR})
 target_link_libraries(codegen PRIVATE libllvm ngraph)
+if(NGRAPH_DISTRIBUTED_ENABLE)
+    add_dependencies(codegen libmlsl)
+    target_include_directories(codegen SYSTEM PUBLIC libmlsl)
+    target_link_libraries(codegen PUBLIC libmlsl)
```

Comment:
I don't think we should link libmlsl to codegen. It does not use the library.

Reply:
Removed